pkg/sql/codegen/tensorflow/codegen.go

```diff
@@ -39,6 +40,9 @@ range: [0,Infinity]`, attribute.IntRangeChecker(newInt(0), nil, false, false)},
 	"train.epoch": {attribute.Int, `[default=1]
 Number of epochs the training will run.
 range: [1, Infinity]`, attribute.IntRangeChecker(newInt(0), nil, false, false)},
+	"train.verbose": {attribute.Int, `[default=0]
+Show verbose logs when training.
+possible values: 0, 1`, attribute.IntChoicesChecker([]int{0, 1})},
```

Comment:
IntChoiceChecker returns an error message like `fmt.Errorf("attribute value out of range(%v), actual: %d", choices, i)`, but it's not clear for the user to find out the attribute is relative to.

Reply:
> IntChoiceChecker returns an error message like `fmt.Errorf("attribute value out of range(%v), actual: %d", choices, i)`, but it's not clear for the user to find out the relative attribute name.

This is an existing problem in other checkers, maybe I can fix this later in another commit

Reply:
@weiguoz I'll try polish the error message in the next PR.